panther_analysis_tool/cmd/configsdk_test.py

```diff
@@ -0,0 +1,280 @@
+import argparse
+import base64
+import enum
+import json
+import logging
+from collections import defaultdict
+from functools import reduce
+from typing import Tuple, Final, Dict, List, Any
+
+from panther_analysis_tool.cmd import config_utils
+
+
+class DetectionType(enum.Enum):
```

Comment:
*nit

I wonder if we should import these types from panther core

https://github.com/panther-labs/panther_core/blob/main/panther_core/rule.py#L40-L41

Reply:
I can. The ones you linked to are string constants though and I was specifically wanting an enum here. So I am not sure it would help at all to import the core ones

Reply:
The idea was to reference what I think is the ultimate source of truth of detection types, but these should never change, so probably not a big deal